src/main/java/org/opensearch/ad/caching/PriorityCache.java

```diff
@@ -169,9 +169,9 @@ public ModelState<EntityModel> get(String modelId, AnomalyDetector detector) {
                         id -> {
                             // reset every 60 intervals
                             return new DoorKeeper(
-                                AnomalyDetectorSettings.DOOR_KEEPER_FOR_CACHE_MAX_INSERTION,
-                                AnomalyDetectorSettings.DOOR_KEEPER_FAULSE_POSITIVE_RATE,
-                                detector.getDetectionIntervalDuration().multipliedBy(AnomalyDetectorSettings.DOOR_KEEPER_MAINTENANCE_FREQ),
+                                TimeSeriesSettings.DOOR_KEEPER_FOR_CACHE_MAX_INSERTION,
```

Comment:
Nice! Not needed for the PR -- but it may be worth separating out what is intended (at this point of time) to be in TimeSeries concept and what is intended to be in AD (as the plugin) concept.

Reply:
yes, agreed